tests/test_reporter.py

```diff
@@ -1,4 +1,5 @@
 from __future__ import print_function
```

Comment:
the import should go after standard license header

Reply:
Okay, I'll change this as well..